src/views/RecruitPage/components/common/Tabs/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { ExtraPart, ExtraTabType, Part, PartExtraType, TabType } from '@src/lib/types/universal';
+import * as S from './style';
+
+type TabBarProps =
+  | {
+      type: 'with-all';
+      selectedTab: ExtraPart;
+      onTabClick(targetTab: ExtraPart): void;
+    }
+  | { type: 'without-all'; selectedTab: Part; onTabClick(targetTab: Part): void };
+
+const allTabs: ExtraTabType[] = [
+  {
+    value: PartExtraType.ALL,
+    label: '전체',
+  },
+  {
+    value: Part.PLAN,
+    label: '기획',
+  },
+  {
+    value: Part.DESIGN,
+    label: '디자인',
+  },
+  {
+    value: Part.ANDROID,
+    label: '안드로이드',
+  },
+  {
+    value: Part.IOS,
+    label: 'iOS',
+  },
+  {
+    value: Part.WEB,
+    label: '웹',
+  },
+  {
+    value: Part.SERVER,
+    label: '서버',
+  },
+];
```

Comment:
suggestion;

`tabs` 배열과 중복되는 요소가 많은 것 같은데 다음과 같이 작성해도 되지 않을까요?
```suggestion
const allTabs: ExtraTabType[] = [
  {
    value: PartExtraType.ALL,
    label: '전체',
  },
  ...tabs,
];
```